tests/src/JIT/Regression/JitBlue/DevDiv_200492/DevDiv_200492.cs

```diff
@@ -0,0 +1,92 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+internal class Program
+{
```

Comment:
It would be nice to have comments here to describe the bug and fix that you are testing for.